### PR TITLE
Show proper message when there are no shipping or payment options

### PR DESCRIPTION
**Pull request by mateuszwozniak**
### Please check if the PR fulfills these requirements ###

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Shows proper message on the checkout page when there are no shipping or payment methods available.

**What is the current behavior? (You can also link to an open issue here)**
There's no information about missing shipping/payment methods atm.

### CODE REVIEW

File: examples/shop-with-blog/client/src/pages/shop/Checkout/ShippingMethodSection.js [hunk 1]
```diff
@@ -63,14 +64,22 @@ class ShippingSection extends React.Component {
...
+              onShippingSelected={this.onShippingSelected}
+            />
+          )}
           <ErrorList errors={errors} />
-          <Button disabled={!this.state.selectedShipping} onClick={this.submitShipping}>
-            Continue
-          </Button>
+          {availableShippingMethods.length > 0 && (
+            <Button disabled={!this.state.selectedShipping} onClick={this.submitShipping}>
+              Continue
```
Comment (Borales) [inline examples/shop-with-blog/client/src/pages/shop/Checkout/ShippingMethodSection.js]: @mateuszwozniak should we translate this one too?

File: packages/falcon-i18n/i18n/nl/translations.json [hunk 2]
```diff
@@ -57,6 +57,21 @@
...
+    "couponCode": "Vouchercode",
+    "applyCouponCode": "Activate",
+    "cancelCouponCode": "Cancel code",
+    "invalidCouponCode": "Invalid coupon code",
+    "checkout": "Checkout",
+    "continueShopping": "Shopping"
+  },
+  "checkout": {
+    "noShippingOptions": "There are no available shipping methods for your shipping address. Please contact shop administrator.",
+    "noPaymentOptions": "There are no available shipping methods for this order. Please contact shop administrator."
```
Comment (Borales) [inline packages/falcon-i18n/i18n/nl/translations.json]:
```suggestion
    "noPaymentOptions": "There are no available payment methods for this order. Please contact shop administrator."
```

File: examples/shop-with-blog/client/src/pages/shop/Checkout/ShippingMethodSection.js (hunk 1 above)
Comment (mateuszwozniak) [reply examples/shop-with-blog/client/src/pages/shop/Checkout/ShippingMethodSection.js]: Done

File: packages/falcon-i18n/i18n/nl/translations.json (hunk 2 above)
Comment (mateuszwozniak) [reply packages/falcon-i18n/i18n/nl/translations.json]: Fixed.

File: packages/falcon-i18n/i18n/en/translations.json [hunk 3]
```diff
@@ -69,6 +69,10 @@
     "checkout": "Checkout",
     "continueShopping": "Shopping"
   },
+  "checkout": {
+    "noShippingMethodsAvailable": "There are no available shipping methods for your shipping address. Please contact shop administrator.",
+    "noPaymentMethodsAvailable": "There are no available shipping methods for this order. Please contact shop administrator."
```
[the remark this answers is not in the document]
Comment (mateuszwozniak) [reply packages/falcon-i18n/i18n/en/translations.json]: Fixed.

**Pull request merged**